cabal-install/src/Distribution/Client/CmdInstall.hs

```diff
@@ -831,37 +854,103 @@ installExes
     warnIfNoExes verbosity buildCtx
 
     installMethod <-
-      flagElim defaultMethod return $
+      flagElim (defaultMethod verbosity) return $
         cinstInstallMethod clientInstallFlags
 
+    return $ InstallExe installMethod installdir mkUnitBinDir mkExeName mkFinalExeName
+
+-- | Can we install any built exe by symlinking/copying it?
```

Comment:
Same as with `installableUnitExes`. Also, typos (were there before):
```suggestion
-- | Can we install and build executable by symlinking or copying it?
```

Reply:
The original actually looks correct to me, aside from the abbreviation. Read "any built exe" as "any executable that was built".

Reply:
That's fair! Could we rephrase it like you say? I can understand what you say but I completely missed what the original was saying...

Reply:
Sure.
```suggestion
-- | Can we install any executable that was built by symlinking or copying it?
```

Reply:
Thanks for the suggestions. I didn't change "exe" to "executable" but I did change "symlinking/copying" to "symlinking or copying" everywhere, including in one user-facing message. I 'lso changed "symlink/copy" to "symlink or copy".